File: MODUL-3/Bahasa Python/PRAK304-2310817120014-ALIYARAFFANAURAAYU.py
Num_3 = int(input())

if Num_3 == 0:
    print("Nol")
elif 1 <= Num_3 <= 9:
    print("Satuan")
elif 10 <= Num_3 <= 19:
    print("Belasan")
elif 20 <= Num_3 <= 99:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: MODUL-3/Bahasa C/PRAK304-2310817120014-ALIYARAFFANAURAAYU.c
#include <stdio.h>

int main ()
{
    int Num_3;
    scanf ("%d", &Num_3);
    if (Num_3 == 0){
        printf("Nol");
    }
    else if (Num_3 >= 1 && Num_3 <= 9){
        printf("Satuan");
    }
    else if (Num_3 >= 10 && Num_3 <= 19){
        printf("Belasan");
    }
    else if (Num_3 >= 20 && Num_3 <= 99){
        printf("Puluhan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    return 0;
}